core.mal

```diff
@@ -83,5 +62,3 @@
             `(~(first form) ~@(rest form) ~x)
             (list form x))
           `(->> (->> ~x ~form) ~@more))))))
-
-nil
```

Comment:
This `nil` is intentional so that the result of doing `load-file` is `nil` instead of whatever happens to be at the end of `core.mal` (in this case `->>`).